promql/lex.go

```diff
@@ -395,6 +405,12 @@ func (l *lexer) errorf(format string, args ...interface{}) stateFn {
 func (l *lexer) nextItem() item {
 	item := <-l.items
 	l.lastPos = item.pos
+
+	// TODO(fabxc): remove for version 1.0.
```

Comment:
I don't see us doing any large breaking changes that justify another 0.x and would like to go straight for 1.0 afterwards.